data/model/gc.py

```diff
@@ -201,6 +189,8 @@ def _purge_repository_contents(repo):
         if not found:
             break
 
+    # TODO(kleesc): Check for orphaned manifests that woudln't be caught be the above
```

Comment:
Can this be a JIRA ticket instead? We never really come back to take care of TODO's